### DISCUSSION TITLE
ML with Python Intro Thread

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#74] [General help]
Hey friends 👋 

Welcome to the ML with Python Learning Group! We're going to be using the [freeCodeCamp Machine Learning with Python Curriculum](https://www.freecodecamp.org/learn/machine-learning-with-python/) and adding in any supplementary resources that we find to be helpful as we move through the course.

We'll use the discussions here to learn together, hold each other accountable, and track our progress. In this thread, let's get to know each other better! 

## Intro Prompt
1. Name
2. Occupation
3. Experience level with ML and Python
4. Random question: What's your favorite thing to do in your free time?

**Comment by sensei100** [comment]
Hi! I'm Jennifer Anton and I'm a senior web developer.  I have some experience with Python, but have not learned anything about ML.  In my free time I enjoy reading.

**Comment by michaeljolley** [comment]
Heyo! I'm Michael. I'm in Developer Relations at Deepgram. I have very little experience with ML and most of my Python experience is in the IoT space. In my free time, I like to read, play music, and hang with my kids.

**Comment by BekahHW** [comment]
👋 I'm Bekah. I'm the Technical Community Builder on the DevRel team at Deepgram. I did a little learning last year when I livestreamed learning TensorFlow.js with Gant Laborde's Learning TensorFlow.js book. I have very little experience with Python, but I'm excited to learn more.

In my free time, I hang with my 4 kids and lift weights 🏋️‍♀️

**Comment by SandraRodgers** [comment]
Hi everyone! I'm Sandra. I'm a Developer Experience Engineer at Deepgram, mainly focused on building and maintaining Deepgram's websites used by developers, such as the docs, api reference, blog, and more. So my background is more in front-end engineering, Javascript/CSS/HTML, and I also love frameworks like Vue and Astro.

I don't have much Python experience. I've just played around with it a bit trying to learn it and using the Deepgram SDK. I also don't have a background in ML, but since I work at Deepgram, I'm trying to learn more about it.

I enjoy all the normal activities - hanging out with friends and family, snuggling with my dogs, eating great food out at restaurants with my friends or watching a good show/ reading a good book. 

Looking forward to learning with you all!

**Comment by briancbarrow** [comment]
Hey! I'm Brian and I'm also a DevEx Engineer at Deepgram. I have minimal Python experience and am trying to get more hands on work with it. No background in ML but am excited to learn more.

In my free time I play disc golf and play video games.

**Comment by dominicduffin1** [comment]
Hi everyone, I'm Dominic! I'm a full-stack developer, mainly working with React and NodeJS. I have worked with Python, mainly in university modules but not much in the past couple of years so it will be good to refresh my skills here. I don't have any experience in ML but have been excited seeing what it can do out in the world so am looking forward to learning more.

In my free time I like reading, board games, walking and riding the public transport network.

**Comment by paceaux** [comment]
Howdy, I'm Frank. I'm a senior front-end Drupal engineer at Red Hat and I've previously been a principal consultant/architect. Most of my experience is in Node, but I have written a decent amount of Python. I've dabbled in TensorFlow once or twice. 

In my free time I like learning languages and linguistics, building furniture, and playing guitar.

**Comment by you-go-nicoco** [comment]
Hi everyone! I'm Nicole. I'm a Developer Support Engineer at Deepgram. 👋🏻 

I have a background in full-stack web development as I attended a bootcamp called App Academy, where I mainly worked with Javascript, Node and Ruby. I don't have too much exposure to Python and ML, but I hope to gain knowledge and expertise as I explore other parts of the tech field and deepen my understanding of core concepts we use here at Deepgram!

I love music, art and all things DIY. Basically any excuse to create something beautiful, and better-yet, useful, I'm there! 💃🏻 

Looking forward to getting to know everyone as we start this journey. 😄

**Comment by Joggyjagz7** [comment]
Hello everyone, I am Ejiro. I'm a software developer and technical writer.
I have some experience with python and machine learning. I am here because I love learning and I need to keep my mind busy 🎉
Looking forward to learning and getting to know everyone here.

**Comment by Se0ngjinKim** [comment]
Hi, I'm Jin. 
I was an engineer for controlling robots many years ago and moved to work for non-technical field.
Now I'm going to move back to IT field and learn python and ML.
Usually, I read books in my free time.
I'm so happy to meet you all!

**Comment by stolinski** [comment]
Hi, I'm Scott.
I'm the owner of Level Up Tutorials (Levelup.video) and cohost of Syntax.fm. I've never done a learn group and wanted to pick up some ML skills so I figured this would be a great opportunity for both. 
I've only written Python via Django and a course on Udacity about 10 years ago. 

I breakdance and hang with my family in my free time.

**Comment by joshlopez96** [comment]
Hello, I'm Josh! 
Developer Support Engineer from Deepgram 
Not very familiar with ML concepts, but really looking forward to adding that alongside my growing knowledge of Python!
I enjoy spending time with my 2 dogs and 2 cats in my free time.

**Comment by BALOGUN-DAVID** [comment]
Hello, I’m Balogun David. 
A Data scientist and analyst. I have a practical experience using python. I am here because I want to up skill myself and learn more from industry expert. Happy to e-meet you all!

**Comment by ineelhere** [comment]
Hello, everyone👋 I'm Indraneel - a software engineer and data-driven researcher with a knack for devOps. 
I have experience with coding in Python and ML from a technical perspective. Here to learn more and dive deeper into the concepts with the community!

**Comment by samedovzaur** [comment]
Hi folks. I'm Zaur Samedov. currently student at 3rd year of Computer Engineering. It's been almost a year working on AI, dealing with ML models, and try to reach to the top every day and passionate as the first day! additionally, I do some sketches and playing on random chess tournaments at free times.

**Comment by dsilvadeepal** [comment]
Hi! I'm Deepal. I'm a tech strategist at Salesforce where I build demos for some of our top customers. I have a background in data analytics and have some self taught experience in ML, however it's been more that 2 years since I worked on it. I'm not very familiar with Python, having had more experience in SQL & R. So I'm here to get back to learning Python & ML in a more collaborative learning approach. In my free time I spend time with my 6 yr old son and spend time gardening.

**Comment by dannyoka** [comment]
Hi! I'm Danny. I'm a Software Engineer at Qlik. I worked on their AutoML product, primarily working on building the frontend and occasionally working on the backend services, but rarely with the data science aspect directly. I have a conversational knowledge of ML and like writing Python, but not a deep technical understanding which I'm hoping to gain here. Aside from coding, my wife and 3 boys keep my time pretty occupied.

**Comment by josencf** [comment]
Oh hello there 😊

I'm Jose, and I work at Deepgram as an ML Developer Advocate! Python is my absolute favorite coding language. I helped teach a Python class back in my college days, and pretty much all of my past AI/ML projects used Python (though I remember doing one little AI algorithm implementation project in Julia). 

In my free time, I write little stories, play piano, and do a lot of (midway decent) dancing. Happy to meet everyone!!

**Comment by dhairyachandra** [comment]
Hello 👋 
I am Dhairya Chandra I am working as Data Scientist. I have 3+ years experience in Python and 1+ year with ML. I my free time I play games on Xbox some of my favorite games 🎮 are Call of duty, Rocket League, Flight Simulator and FIFA.

My Portfolio: [medhairya.com](https://www.medhairya.com)

**Comment by nickytonline** [comment]
Hi everyone! My name is Nick Taylor. I'm currently a Staff Software Engineer at Netlify working on frameworks in the ecosystem team.

I have no experience with Python aside from one job where I was converting Django to React. In terms of ML, no experience really.

In my free time, I enjoy working out, snowboarding, and OSS.

You can find me on the web at [iamdeveloper.com](https://iamdeveloper.com)

**Comment by Azmaeus** [comment]
Hi, I'm Eric. A little late to the party but excited to catch up! I'm a Software Engineer with some professional background in Java and C#, educational background in C++, and some hobby background in Python, Javascript, Powershell, vbscript, and others. I have no experience in ML yet, and it's been a couple years since I used Python so I'll need to shake off some rust. In my free time my favorite thing to do is spend time with my kids (usually video games), but I do love a good game of Pickleball which I play daily.
